etc/cvmfs/osgstorage-auth.conf

```diff
@@ -48,62 +48,13 @@ CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://xcachevirgo.pic.es:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ligo.hpc.swin.edu.au:8443/"
 
-
-distroversion()
-{
-    # systemd-based systems have /etc/os-release
-    # Prefer /etc/os-release because lsb_release isn't always installed
-    #   on systems that have /etc/os-release
-    if [ -f /etc/os-release ]; then
-        # If this function is moved back under etc/cvmfs so it is
-        #   parsed by cvmfs, the '=' will confuse the parser in
-        #   cvmfs versions <= 2.8.1.  Note that under cvmfsexec
-        #   this function does get parsed.
-        sed -n 's/"//g;s/^VERSION_ID=//p' /etc/os-release
-    else
-        lsb_release -rs
-    fi
-}
 CVMFS_AUTHZ_CVMFS_TOKEN_VARNAME=BEARER_TOKEN_FILE
-CVMFS_X509_AUTHZ_LIBS_VALID=false
-if /usr/libexec/cvmfs/authz/cvmfs_x509_validator 2>/dev/null; then
-    CVMFS_X509_AUTHZ_LIBS_VALID=true
-fi
-if [ ! -f /etc/grid-security/certificates/cilogon-basic.pem ] || \
-      [ ! -f /etc/vomses ] || [ ! -d /etc/grid-security/vomsdir/fermilab ] || \
-        ! $CVMFS_X509_AUTHZ_LIBS_VALID; then
-    # Get at least CA certs and voms info files from the oasis repo
-
-    OSGMISDIR="$CVMFS_MOUNT_DIR/oasis.opensciencegrid.org/mis"
-    OSGBASEDIR="$OSGMISDIR/osg-wn-client"
-    X509_CERT_DIR=$OSGMISDIR/certificates
-    if $CVMFS_X509_AUTHZ_LIBS_VALID; then
-        # the needed libraries are already installed
-        # we'll only use portable files, so pick any osg-wn-client
-        CVMFS_AUTHZ_OSG_BASE_DIR="$OSGBASEDIR/current/el8-x86_64"
-    else
-        # we'll use both libraries and portable files
-        # only Redhat Enterprise Linux or derivatives are supported
-        DISTROVERSION="`distroversion|cut -d. -f1`"
-        DISTROARCH="`arch`"
-        # OSGRELEASE=current
-        # choose the release that includes GSI libraries, until LIGO
-        #   is completely switched over to tokens
-        OSGRELEASE=3.5/current
-        if [ "$DISTROVERSION" = 6 ]; then
-            # choose the last OSG release to support el6
-            OSGRELEASE=3.4/current
-        fi
-        CVMFS_AUTHZ_OSG_WN_CLIENT="$OSGBASEDIR/$OSGRELEASE/el$DISTROVERSION-$DISTROARCH"
-    fi
+if [ ! -f /etc/grid-security/certificates/cilogon-basic.pem ]; then
+    # Get CA certs from the oasis repo
+    X509_CERT_DIR="$CVMFS_MOUNT_DIR/oasis.opensciencegrid.org/mis/certificates"
 else
     # this is needed because cvmfs would otherwise set this to an empty
     #  string, and cvmfs-2.3.2 tries to use the empty string as its CA dir
     X509_CERT_DIR=/etc/grid-security/certificates
 fi
-# The wrapper scripts here are mostly needed because LD_LIBRARY_PATH can't
-#  be exported.  See https://sft.its.cern.ch/jira/browse/CVM-1023
-# Other variables with a CVMFS_AUTHZ_ prefix will be passed to the helper
-#  with the prefix stripped off.
-CVMFS_AUTHZ_SEARCH_PATH="$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/libexec/authz"
-
+CVMFS_AUTHZ_SEARCH_PATH="$CVMFS_DIST/usr/libexec/cvmfs/authz"
```

Comment:
Is setting this variable still necessary?

Reply:
It is.  I tried without it and it didn't work.

Reply:
Ah but you're right, it does work if I also remove the `CVMFS_AUTHZ_SEARCH_PATH=` settings.  Removed.